libs/template/materialize.go

```diff
@@ -104,6 +104,11 @@ func Materialize(ctx context.Context, configFilePath, templateRoot, outputDir st
 
 // If the given templateRoot matches
 func prepareBuiltinTemplates(templateRoot string, tempDir string) (string, error) {
+	// User specified CWD. They do not want to use any built-in templates.
+	if templateRoot == "." {
+		return templateRoot, nil
+	}
+
```

Comment:
There is more restrictive constraint to fix this once-and-for-all.

Without this, the parameter `./mlops-stacks` will still resolve the built-in template.

```go
builtinPath := path.Join("templates", templateRoot)
// Check that `templateRoot` is a clean basename, i.e. `some_path` and not `./some_path`.
if path.Base(templateRoot) != templateRoot {
  return templateRoot, nil
}
```

Reply:
Done